keep-ui/app/alerts/alerts.tsx

```diff
@@ -237,7 +239,59 @@ export default function Alerts({
     );
   };
 
-  const setAssignee = (fingerprint: string, lastReceived: Date, unassign: boolean) => {
+  const AlertTableTabPanel = ({ preset }: { preset: Preset }) => {
+    const [rowSelection, setRowSelection] = useState<RowSelectionState>({});
+
+    const amountOfRowSelections = Object.keys(rowSelection).length;
+
+    return (
+      <TabPanel className="mt-4">
+        {amountOfRowSelections ? (
+          <AlertActions
+            amountOfRowSelections={amountOfRowSelections}
+            onDelete={onDelete}
+            alerts={currentStateAlerts.slice(startIndex, endIndex)}
```

Comment:
I think we're always passing all the alerts in the sliced view here so it deletes everything in the same view, although I only selected 1 alert.

Reply:
Should be resolved with https://github.com/keephq/keep/pull/657/commits/2b6fbde2df1e14f23f30be9e05166a8902fc30b8. That was a brain fart on my end 🤯